rdmo/management/assets/js/components/element/Catalog.js

```diff
@@ -26,12 +26,15 @@ const Catalog = ({ config, catalog, elementActions, display='list',
 
   const toggleAvailable = () => elementActions.storeElement('catalogs', {...catalog, available: !catalog.available })
   const toggleLocked = () => elementActions.storeElement('catalogs', {...catalog, locked: !catalog.locked })
-
+  const toggleCurrentSite = () => elementActions.storeElement('catalogs', catalog, null, 'toggle-site')
+  let has_current_site =  config.settings.multisite ? catalog.sites.includes(config.currentSite.id) : true
```

Comment:
this should be `hasCurrentSite`, be `const` and be moved further up where the variables are.

Reply:
I think it would be even better to do this inline and have one prop per line for `ToggleCurrentSiteLink`.

Reply:
Ive moved it to the inline props